### DISCUSSION TITLE
MQTT motion on Frigate startup

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#9675] [General]
I have two cameras monitoring my sump pumps in the basement. When they start up there is enough movement to trigger Frigate motion. Node Red listens for "Sump1/motion" and "Sump2/motion" and announces via Google Home that sump pump is running.
Since installing Frigate .13.1 MQTT reports "motion" on both Sump1 and Sump2 cameras during **Frigate startup or restart.**

**Comment by NickM-27** [comment]
this can occur while the motion is calibrating. You may just want to look in to adding a timeout from when frigate first starts

**Reply by 18923** [reply]
It's not really a problem in my case. Just thought I mention it.
Nick, thank you for the quick response.
